api-docs/rst/dev-guide/api-operations/methods/get-get-scaling-group-state-v1.0-tenantid-groups-groupid-state.rst

```diff
@@ -14,14 +14,27 @@ This operation retrieves the current state of a scaling group.
 The *GroupState* object consists of the following properties:
 
 
-
-*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value is set to true, the group will not scale up or down. All policy execution calls are ignored while this value is set to true.
+*  *paused*. If ``paused=TRUE``, the group does not scale up or down. All
+   scheduled or API-generated policy operations are suspended, and convergence
+   is not triggered. When the group is paused, any POST requests to
+   :ref:`converge <trigger-convergence>` or
+   :ref:`execute policy <execute-policye>`
+   operations return a ``403 GroupPausedError`` response.
+   If ``paused=FALSE``, all group scaling and convergence operations resume and
```

Comment:
is not triggered. When the group is paused, any POST requests to
+   :ref:`converge <post-converge-v1.0-tenantid-groups-groupid>` or
+   :ref:`execute policy <post-execute-policy-v1.0-tenantid-groups-groupid-policies-policyid-execute>`
+   operations return a ``403 GroupPausedError`` response.